make/migrations/postgresql/0130_2.10.0_schema.up.sql

```diff
@@ -1,3 +1,5 @@
 DROP TABLE IF EXISTS harbor_resource_label;
 
-CREATE INDEX IF NOT EXISTS idx_artifact_accessory_subject_artifact_id ON artifact_accessory (subject_artifact_id);
\ No newline at end of file
+CREATE INDEX IF NOT EXISTS idx_artifact_accessory_subject_artifact_id ON artifact_accessory (subject_artifact_id);
+
+ALTER TABLE artifact ADD COLUMN artifact_type varchar(255);
```

Comment:
do we need to migrate the existing data?

Reply:
Will update this later